android/KMEA/app/src/main/java/com/tavultesoft/kmea/packages/LexicalModelPackageProcessor.java

```diff
@@ -0,0 +1,228 @@
+package com.tavultesoft.kmea.packages;
+
+import android.util.Log;
+
+import com.tavultesoft.kmea.KMManager;
+import com.tavultesoft.kmea.util.FileUtils;
+import com.tavultesoft.kmea.util.ZipUtils;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An extension of PackageProcessor
+ * A class of static methods for use in handling Keyman's .kmp file format, as it relates to the
+ * KMEA Engine. This is primarily for installing lexical model packages.
+ */
+public class LexicalModelPackageProcessor extends PackageProcessor {
```

Comment:
If all the class's methods are static, then why extend `PackageProcessor`?  By default, `LexicalModelPackageProcessor` should still be able to access all class members, since the two live in the same namespace.  (No access modifier = 'internal', which is `protected` + 'within namespace'.)

Reply:
Addressed in refactoring below